ddl/ddl.go

```diff
@@ -540,6 +543,7 @@ func (d *ddl) doDDLJob(ctx sessionctx.Context, job *model.Job) error {
 		// If a job is a history job, the state must be JobSynced or JobCancel.
 		if historyJob.IsSynced() {
 			log.Infof("[ddl] DDL job ID:%d is finished", jobID)
+			ctx.GetSessionVars().StmtCtx.AddAffectedRows(uint64(historyJob.GetRowCount()))
```

Comment:
Watch out, add index also use `job.RowCount`. So this will make `add index` return xx row affected.
In Mysql:
```sql
mysql root@127.0.0.1:test> create table t1 (a int);
Query OK, 0 rows affected
Time: 0.010s
mysql root@127.0.0.1:test> insert into t1 values (0),(1),(2),(3),(4),(5);
Query OK, 6 rows affected
Time: 0.001s
mysql root@127.0.0.1:test> alter table t1 add index idx(a);
Query OK, 0 rows affected
Time: 0.008s
mysql root@127.0.0.1:test> create table t2 select a from t1;
Query OK, 6 rows affected
Time: 0.007s
```
In This PR:
```sql
mysql root@127.0.0.1:test> create table t1 (a int);
Query OK, 0 rows affected
Time: 0.008s
mysql root@127.0.0.1:test> insert into t1 values (0),(1),(2),(3),(4),(5);
Query OK, 6 rows affected
Time: 0.001s
mysql root@127.0.0.1:test> alter table t1 add index idx(a);
Query OK, 6 rows affected
Time: 0.016s
mysql root@127.0.0.1:test> create table t2 select a from t1;
Query OK, 6 rows affected
Time: 0.012s
```

Reply:
Sure, thank you!